globals.d.ts

```diff
@@ -1,19 +1,21 @@
 declare namespace NodeJS {
   export interface ProcessEnv {
-    /* RPC URL for Hardhat Network forking, required for running tests on mainnet fork with tracing */
-    MAINNET_FORKING_URL?: string;
+    /* iternal logging verbosity (used in scratch deploy / integration tests) */
+    LOG_LEVEL?: "all" | "debug" | "info" | "warn" | "error" | "none"; // default: "info"
 
-    /* logging verbosity */
-    LOG_LEVEL?: "all" | "debug" | "info" | "warn" | "error" | "none";
+    /**
+     * Flags for changing the behavior of the Hardhat Network
+     */
+
+    /* RPC URL for Hardhat Network forking, required for running tests on mainnet fork with tracing */
+    HARDHAT_FORKING_URL?: string;
 
     /**
      * Flags for changing the behavior of the integration tests
      */
 
     /* if "on" the integration tests will deploy the contracts to the empty Hardhat Network node using scratch deploy */
-    INTEGRATION_SCRATCH_DEPLOY?: "on" | "off";
-    /* if "on" the integration tests will enable assertions and checks for the simple DVT module */
-    INTEGRATION_SIMPLE_DVT_MODULE?: "on" | "off";
```

Comment:
Hmm, the template has an optionality to deploy sDVT or not, while here, we exclude it deliberately. Not sure if this is consistent.

Reply:
The template can also be used to deploy on testnets, and in that case, the step with SDVT may be skipped. However, local scratch deployment now always enables the SDVT module to test the state with 2 modules. This is why I decided to remove the bias for running with or without SDVT, since on the mainnet it always runs with SDVT enabled. But this is a good catch! I'm not solid in my decision here.